Assets/Scripts/Layers/Properties/PropertyPanels/MTLImportPropertySection.cs

```diff
@@ -0,0 +1,60 @@
+using Netherlands3D.Twin.Layers;
+using UnityEngine;
+
+namespace Netherlands3D.Twin
+{
+    public class MTLImportPropertySection : MonoBehaviour
+    {
+        [SerializeField] private GameObject defaultImportPanel;
+        [SerializeField] private GameObject hasMtlPanel;
+        [SerializeField] private GameObject importErrorPanel;
+        public ObjSpawner ObjSpawner { get; set; }
+        
+        private void Start()
+        {
+            SetNormalUIPanels();
+            ObjSpawner.MtlImportSuccess.AddListener(OnMTLImportError);
+        }
+
+        //called in the inspector by FileOpen.cs
+        public void ImportMTL(string path)
+        {
+            if (path.EndsWith(','))
```

Comment:
gaat dit ook goed als de string csv,obj,mtl,json bijvoorbeeld heeft?

Reply:
het probleem is dat de file importer altijd een `,` op het eind zet dus die moet eraf om het te kunnen verwerken.

Reply:
en momenteel verwacht dit script 1 mtl file, dus meerdere files werkt niet, maar de file opener is ingesteld zonder multiselect en met mtl als extensie